blockchain/votebits.go

```diff
@@ -110,24 +110,20 @@ func (c deploymentChecker) Condition(node *blockNode) ([]thresholdConditionTally
 	// Setup tally array and iterate over Choices to assemble the vote
 	// information into the thresholdConditionTally array.
 	tally := make([]thresholdConditionTally, len(c.deployment.Vote.Choices))
-	for _, v := range node.voteBits {
-		// Make sure only valid bits are set.
-		x := c.deployment.Vote.Mask & v
-		isIgnore, err := c.deployment.Vote.IsIgnore(x)
-		if err != nil {
-			// Ignore invalid vote; others may be ok.
-			continue
-		}
-		isNo, err := c.deployment.Vote.IsNo(x)
-		if err != nil {
-			// Ignore invalid vote; others may be ok.
+	for t, vote := range c.deployment.Vote.Choices {
```

Comment:
This is iterating choices not votes, so I think the variable should be named choice.